geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -8631,7 +8630,9 @@ public void clearRegionLocally(RegionEventImpl regionEvent, boolean cacheWrite,
   @Override
   void basicLocalClear(RegionEventImpl rEvent) {
     getDataView().checkSupportsRegionClear();
+    final long startTime = startClear();
     cmnClearRegion(rEvent, false/* cacheWrite */, false/* useRVV */);
```

Comment:
the cmnClearRegion call should be in a try and the endClear should be in a finally

Reply:
done